### ISSUE TITLE
기동어 이후 UI 구현

### ISSUE THREAD

**Issue by jujuwon** [Volvo-Voice-Assistance/assistance-android#2]
Voice Assistance 가 기동어를 인식해 입력을 대기 중인 상태가 되었을 때,
UI 적으로 가시화할 방법과 표현할 UI 를 구현합니다.
- 백그라운드 상에서 디스플레이를 제어하여 입력 대기 중인 UI 가 표시되도록 구현합니다.

**Comment by jujuwon** [comment]
@Moon-janggon 담당자 변경했습니다 ! 이어서 구체화하시고 이 이슈에서 구현까지 이어가시면 됩니다 !

**Comment by Moon-janggon** [comment]
1.Service 에서 WindowManager 를 이용해 최상단 View 띄우기 기능을 검토해 보았습니다
사용 가능한 것으로 확인되어 컴파일은 진행 되나 가상머신에서 구동시
권한 문제관련 오류로 해결중에 있습니다.

![image](https://user-images.githubusercontent.com/80737840/232315298-be44deeb-9353-4130-b68d-b99a343899c0.png)

백그라운드에서 기동어 인식시 이미지와 같이 아이콘이 화면 최상단에 뜨는 형식으로 구동 될 것으로 예상되며 
이에 대해 괜찮으신지 의견 부탁드리겠습니다.
@KDW03 

2.FloatingView는 4/17 월요일 회의 이후 답변을 받고 진행 여부를 확인하는 걸로 알고 있는데 아니라면 회신 부탁드리겠습니다.
@jujuwon

**Comment by jujuwon** [comment]
> 2.FloatingView는 4/17 월요일 회의 이후 답변을 받고 진행 여부를 확인하는 걸로 알고 있는데 아니라면 회신 부탁드리겠습니다. @jujuwon

@Moon-janggon 맞습니다 ! 하지만 일단 FloatingView 라이브러리 없이 안드로이드 기본 WindowManager 로 구현 가능하다면,
1번 방식으로 구현해주시면 될 듯 싶습니다.
기동어 이후 View 를 띄우고, 사용자가 명령어를 입력하는 동안 듣고 있다는 것을 알 수 있도록 View 가 반응하고,
명령어 인식 성공/실패 에 따라 최상위 View 에서 Toast 메시지를 띄우거나, 앱 View 를 띄우도록 하는 게 가능한지 확인하고
그대로 구현해주시면 될 듯 싶습니다.
이에 대한 진행 상황 내일 공유해주세요 🙋‍♂️

**Comment by Moon-janggon** [comment]
![스크린샷 2023-04-17 183557](https://user-images.githubusercontent.com/80737840/232445931-8618f38b-3b73-4943-bbbb-bdbd8a37c3aa.png)

이미지와 같이 구현 했습니다.  이후 음성인식 상태 일때 변환까지 구현 하고자 하는데
진행방향 문제 없는지 또는 아이콘 배치나 기타 의견 있으시면 말씀 해주세요 
@jujuwon 
@KDW03

**Comment by jujuwon** [comment]
@Moon-janggon 피그마 UI 와 동일하게 만들어주세요 !

**Comment by Moon-janggon** [comment]
![스크린샷 2023-04-17 235733](https://user-images.githubusercontent.com/80737840/232525099-3c1f0222-e5d8-4c9e-aaf1-28963de524bb.png)

![스크린샷 2023-04-17 235850](https://user-images.githubusercontent.com/80737840/232525270-46244397-1c55-49b5-93f4-1b1a492b4978.png)

배치 변경과 기동어 입력 시 변환 상태 입니다 현재는 _onClickListener_로 구현되어 있고 tts 전달 구조에 따라 변경 하겠습니다.

**Comment by jujuwon** [comment]
@KDW03 작업 결과물 합칠 수 있게 dev 브랜치와 각 feature 브랜치 생성 부탁드려요 !
feature 중 해당 UI 구현 브랜치 @Moon-janggon 에게 전달해주시고,
기동어 및 명령어 인식 시 해당 UI 기동되는 부분 역시 전달해서 실제 구현으로 이어질 수 있게 가이드 부탁드립니다 !
